Python/generated_cases.c.h

```diff
@@ -20,15 +20,17 @@
             _PyStackRef rhs;
             _PyStackRef res;
             // _SPECIALIZE_BINARY_OP
-            rhs = stack_pointer[-1];
-            lhs = stack_pointer[-2];
             {
+                rhs = stack_pointer[-1];
```

Comment:
The assignments being inside the braces is largely a consequence of making the analysis flow sensitive.
I could attempt to move them back, but I worry that it would contort the code just to minimize the diff for this one PR.

It will take me a lot longer to make the change than your browser to open the file.